### ISSUE TITLE
Fullscreen workspace switching is strange

### ISSUE THREAD

**Issue by KyleUltimate** [hyprwm/Hyprland#439]
https://user-images.githubusercontent.com/68589851/181502442-00ca2401-afc2-41b0-97d9-1923e1503268.mp4

It's... a bit strange

**Comment by KyleUltimate** [comment]
```conf
animations {
    enabled=1
    animation=workspaces,1,6,my_curve
}
```
animation for workspace moves

**Comment by vaxerski** [comment]
huh. weird. May I get your config too?

**Comment by KyleUltimate** [comment]
It's separated into different files, so there you go!
`decoration.conf` https://0x0.st/ojGN.conf
`general.conf` https://0x0.st/ojGb.conf
`keybinds.conf` https://0x0.st/ojGc.conf
`rules.conf` https://0x0.st/ojGT.conf
`startup.conf` https://0x0.st/ojGA.conf
`hyprland.conf`https://0x0.st/ojGm.conf (just sourcing stuff)

**Comment by vaxerski** [comment]
should be fixed with 6d35d642ea40d365ca81e5e0c28429fc5a9b1b8f

**Comment by KyleUltimate** [comment]
Yep it's fixed!
